subjects/devops/play-with-containers/audit/README.md

```diff
@@ -1,223 +1,223 @@
-
-###### Did the student reply correctly to the questions?
-
-#### Check the Docker images:
-
-```console
-user:~$ docker images
-REPOSITORY              TAG          IMAGE ID       CREATED        SIZE
-inventory-database     <...>          <...>         <...>          <...>
-billing-database       <...>          <...>         <...>          <...>
-inventory-app          <...>          <...>         <...>          <...>
-billing-app            <...>          <...>         <...>          <...>
-RabbitMQ               <...>          <...>         <...>          <...>
-api-gateway-app        <...>          <...>         <...>          <...>
-user:~$
-```
-
-###### Is there a Docker image for each service with the same service name?
-
-##### Ask the following questions to the group or student
-
-##### What is a Docker image?
-
-##### Why do we use Docker images?
-
-##### Where can we find some public Docker images?
-
-###### Did the student reply correctly to the questions?
-
-#### Inventory API Endpoints
-
-##### Open Postman and make a `POST` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/movies/` address with the following body as `Content-Type: application/json`:
-```json
-{
-    "title": "A new movie",
-    "description": "Very short description"
-}
-```
-
-###### Can you confirm the response was the success code `200`?
-
-##### In Postman make a `GET` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/movies/` address.
-
-###### Can you confirm the response was success code `200` and the body of the response is in `json` with the information of the last added movie?
-
-#### Billing API Endpoints
-
-##### Open Postman and make a `POST` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/billing/` address with the following body as `Content-Type: application/json`:
-```json
-{
-    "user_id": "20",
-    "number_of_items": "99",
-    "total_amount": "250"
-}
-```
-
-###### Can you confirm the response was success code `200`?
-
-##### Stop the billing-app container
-
-###### Can you confirm the `billing-app` container was correctly stopped?
-
-##### Open Postman and make a `POST` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/billing/` address with the following body as `Content-Type: application/json`:
-```json
-{
-    "user_id": "22",
-    "number_of_items": "10",
-    "total_amount": "50"
-}
-```
-
-###### Can you confirm the response was success code `200` even if the `billing_app` is not working?
-
-#### Bonus
-
-###### +Did the student add any optional bonus?
-
-###### +Is this project an outstanding project?
+#### General
+
+##### Check the Repo content
+
+A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.
+
+###### Are all the required files present?
+
+###### Was the `.env` file excluded from the git files?
+
+###### Are all pushed files in the repo clean of any credentials or passwords?
+
+##### Ask the following questions to the group or student:
```

Comment:
we should group the questions like in the roleplay for the other projects, and specify the number of correct answers needed to pass the question, or say "all"